site/docs/spec.md

```diff
@@ -935,14 +936,16 @@ Table metadata is serialized as a JSON object according to the following table.
 |**`location`**|`JSON string`|`"s3://b/wh/data.db/table"`|
 |**`last-updated-ms`**|`JSON long`|`1515100955770`|
 |**`last-column-id`**|`JSON int`|`22`|
-|**`schema`**|`JSON schema (object)`|`See above`|
+|**`schema`**|`JSON schema (object)`|`See above, read schemas instead`|
+|**`schemas`**|`JSON schemas (list of objects)`|`See above, read schemas instead`|
```

Comment:
might be able to omit "read schemas instead"?

Reply:
Yes.